common/network-common/src/main/java/org/apache/spark/network/protocol/Message.java

```diff
@@ -39,6 +39,9 @@
     StreamRequest(6), StreamResponse(7), StreamFailure(8),
     OneWayMessage(9), User(-1);
 
+    /** Encoded length in bytes. */
+    public static final int LENGTH = 1;
```

Comment:
It is made to avoid using hardcoded values in a very different place in TransportFrameDecoder. Where I have Message.Type object before decoding the type but the static member can be accessed.